nest-forms-backend/src/convert/convert.service.ts

```diff
@@ -282,30 +183,28 @@ export default class ConvertService {
       return this.generateTaxPdf(jsonForm, formId)
     }
 
-    const pdfData: PdfPreviewDataResponseDto = {
-      formDefinitionSlug: formDefinition.slug,
-      jsonForm,
-      serverFiles: form.files,
-      additionalMetadata,
+    let pdfBuffer: Buffer
+    try {
+      pdfBuffer = await renderSummaryPdf({
+        jsonSchema: formDefinition.schemas.schema,
+        uiSchema: formDefinition.schemas.uiSchema,
+        formData: jsonForm as GenericObjectType,
+        launchBrowser: () => chromium.launch(),
+        clientFiles,
+        serverFiles: form.files,
+      })
+    } catch (error) {
+      throw this.throwerErrorGuard.InternalServerErrorException(
```

Comment:
Can we maybe log some more specific error, instead of just `Error 500: PDF generation failed`?